src/type_filter.rs

```diff
@@ -84,6 +85,7 @@ where
     pub is_in: Option<Vec<T>>,
     pub is_not_in: Option<Vec<T>>,
     pub is_null: Option<bool>,
+    pub is_not_null: Option<bool>,
```

Comment:
A strange question. What if one supply `false` for these two fields?

```graphql
{
  customer(
    filters: { active: {
      # Normally...
      is_null: true,
      is_not_null: true,
      # How about...?
      is_null: false,
      is_not_null: false,
    } }
  ) {
    ...
  }
}
```

Reply:
Should be fixed.